src/store/gateway/ethereum.ts

```diff
@@ -380,27 +382,35 @@ export async function ethereumDeposit(context: ActionContext, funds: Funds) {
           funds.weiAmount,
           context.rootState.ethereum.address,
         )
-        fb.showLoadingBar(false)
         fb.showSuccess(i18n.t("components.gateway.deposit.confirmed").toString())
       } catch (err) {
-        fb.showLoadingBar(false)
-        if ("imToken" in window) {
+        let sendToSentry = true
+        // NOTE remove this when BSC supports websocket
+        if (isBCWallet(context.rootState.ethereum.wallet!) &&
```

Comment:
`isBCWallet()` should handle `null/undefined` appropriately, so the `!` isn't necessary at the call sites.